### PR TITLE
Add scripts for Jenkins and update POM info in build.gradle

### PR DESCRIPTION
**Pull request by pasin**
Instruction for Jenkins to build CBL-Java: https://gist.github.com/pasin/a61b8d43aac96f3bd88048f5717ddaca

Some of the scripts might need to be changed if we restructure couchbase-lite-java repo.

### CODE REVIEW

File: etc/jenkins/build_linux.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+fi
+
+EDITION="$2"
+if [ -z "$EDITION" ]; then
+    usage
+fi
+
+touch local.properties
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```
Comment (bmeike) [inline etc/jenkins/build_linux.sh]: really minor point:  "SCRIPT_DIR" seems like an odd name, given that $CBL_JAVA_DIR/scripts is where all the scripts are...

Comment (pasin) [reply etc/jenkins/build_linux.sh]: It automatically determine where the script is located so the script can be run from any location.

Comment (bmeike) [reply etc/jenkins/build_linux.sh]: Yeah!  It works fine... I just thought the name was a little odd, given that the scripts that you run (which might be said to be in a scripts directory) were somewhere else!  No worries.  It's fine.

**Comment by pasin** [comment]
The PR validation is now passed after merging the PR in java-ee.

**Pull request merged**